### ISSUE TITLE
requirement-checker directory does not exist

### ISSUE THREAD

**Issue by rask** [humbug/box#153]
I have installed box as a global binary via

    $ composer global require humbug/box:^3.0@dev

and when I run

    $ box compile

inside a project directory I get the following error:

```
? Adding requirements checker

In Finder.php line 547:      
                                                                                                                      
  The "/home/<user>/.config/composer/vendor/humbug/box/src/RequirementChecker/../../.requirement-checker" directory does not exist.
```

Box has worked before so I presume this has something to do with the project being initially built with box 2 and now version 3 has changed something. My `box.json` is as such:

```
{
    "main": "bin/mybin",
    "directories": ["src"],
    "finder": [
        {
            "name": "*.php",
            "exclude": ["tests"],
            "in": "vendor"
        }
    ],
    "git-version": "release_version",
    "output": "build/mybin.phar",
    "stub": true
}
```

This same error appears locally as well as in our CI tooling. I will disable requirements checking until I can find a solution for this. :)

**Comment by theofidry** [comment]
Closed via #155, I'll do a release tonight

**Comment by rask** [comment]
Missed the earlier issue you created about this. Thanks!
